Source/Core/Core/HW/DSP.h

```diff
@@ -56,6 +56,8 @@ union UDSPControl
 	UDSPControl(u16 _Hex = 0) : Hex(_Hex) {}
 };
 
+extern UDSPControl g_dspState;
```

Comment:
I'm not a big fan of that, but I don't really know how it could be made better... :(